react/week-2/lesson.md

```diff
@@ -493,26 +495,37 @@ useEffect(() => {
 And here is a more complete example:
 
 ```js
-import React, { useState, useEffect } from 'react'; // remember to import the Hook(s) you need!
+import React, { useState, useEffect } from "react"; // remember to import the Hook(s) you need!
 
 function MartianPhotoFetcher() {
-  const [marsPhotos, setMarsPhotos] = useState();
+  const [marsPhotos, setMarsPhotos] = useState({});
 
   useEffect(() => {
-    fetch(`https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos?earth_date=2015-6-3&api_key=DEMO_KEY`)
+    fetch(
+      `https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos?earth_date=2015-6-3&api_key=DEMO_KEY`
+    )
       .then(res => res.json())
       .then(data => setMarsPhotos(data));
-    });
   }, []);
 
   return (
     <div>
-      // TODO: update this example
+      {marsPhotos.photos &&
+        marsPhotos.photos.map((photo, index) => {
+          return (
+            <img
+              key={`mars-photo-${index}`}
```

Comment:
just curious -  why not just {index for the key}?  that'd be just as unique in context and react doesn't care what it is.

Reply:
True it would be unique in this context. I thought I'd slip in a more realistic example of what often needs to be done to get a unique key in a larger app, where there might be more than one `map` on a page and therefore some duplicate indexes. 
Happy either way here, can revert to `key={index}` if we think it's clearer.